feather/server/src/entities.rs

```diff
@@ -20,12 +20,20 @@ impl SpawnPacketSender {
 /// when to send movement updates.
 #[derive(Copy, Clone, Debug)]
 pub struct PreviousPosition(pub Position);
+#[derive(Copy, Clone, Debug)]
+pub struct PreviousOnGround(pub OnGround);
 
 pub fn add_entity_components(builder: &mut EntityBuilder, init: &EntityInit) {
     if !builder.has::<NetworkId>() {
         builder.add(NetworkId::new());
     }
-    builder.add(PreviousPosition(*builder.get::<Position>().unwrap()));
+
+    let prev_position = *builder.get::<Position>().unwrap();
+    let on_ground = *builder.get::<OnGround>().unwrap();
```

Comment:
Please either remove the unwraps, add a comment explaining why they're safe to do, or document that the function panics.